### ISSUE TITLE
Dataset Hosting

### ISSUE THREAD

**Issue by marlens123** [RolnickLab/snowgalileo#23]
- [x] Host initial version
- [x] Adapt code to work with the publishing format

**Comment by liellnima** [comment]
- [ ] Zenodo
- [ ] License: Restrictions from enveo / DLR?
- [ ] Inputs: only reproducible via google earth engine
- [ ] Provided through hosting: lon/lats for pre-train + masks for eval inputs + outputs ("ground truth in swtizerland: landsat + sentinel 2 + 3")

**Comment by liellnima** [comment]
Celia: Providing hosting through ESA??
